### PR TITLE
Add DebtLocker + Loan V3 packages

### PR DESCRIPTION
**Pull request by Jakkc**
|       Type        |              Ticket              |
| :---------------: | :------------------------------: |
| Feature | [Link](https://app.shortcut.com/maplefinance/story/6462/maple-js-add-debtlocker-v3-loan-v3-to-sdk) |

## Problem
- SDK needs LoanV3 & DebtLockerV3 packages.

## Solution
- Add packages to config.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -140,4 +144,4 @@
       ]
     }
   }
-}
+}
```
Comment (michaelsmueller) [inline config.json:147]: add space

File: package.json [hunk 2]
```diff
@@ -57,4 +59,4 @@
     "shelljs": "^0.8.5",
     "typechain": "^7.0.0"
   }
-}
+}
```
Comment (michaelsmueller) [inline package.json:62]: add space

File: package.json [hunk 3]
```diff
@@ -38,12 +38,14 @@
...
     "@maplelabs/debtLockerV1": "npm:@maplelabs/debt-locker@v1.0.0",
     "@maplelabs/debtLockerV2": "npm:@maplelabs/debt-locker@v2.0.0",
+    "@maplelabs/debtLockerV3": "npm:@maplelabs/debt-locker@v3.0.0",
     "@maplelabs/environment-mocks": "npm:@maplelabs/environment-mocks@v1.0.0",
     "@maplelabs/fundingLocker": "npm:@maplelabs/funding-locker@v1.0.0",
     "@maplelabs/lateFeeCalculator": "npm:@maplelabs/late-fee-calculator@v1.0.0",
     "@maplelabs/liquidityLocker": "npm:@maplelabs/liquidity-locker@v1.0.0",
     "@maplelabs/loanV1": "npm:@maplelabs/loan@v1.0.0",
     "@maplelabs/loanV2": "npm:@maplelabs/loan@v2.0.0",
+    "@maplelabs/loanV3": "npm:@maplelabs/loan@v3.0.0",
```
Comment (michaelsmueller) [inline package.json:48]:
```suggestion
    "@maplelabs/loanV3": "`npm:@maplelabs/loan@v3.0.0-rc.0",
```

Comment (michaelsmueller) [inline package.json:48]:
```suggestion
    "@maplelabs/loanV3": "npm:@maplelabs/loan@v3.0.0-rc.0",
```

File: package.json [hunk 4]
```diff
@@ -38,12 +38,14 @@
     "@maplelabs/collateralLocker": "npm:@maplelabs/collateral-locker@v1.0.0",
     "@maplelabs/debtLockerV1": "npm:@maplelabs/debt-locker@v1.0.0",
     "@maplelabs/debtLockerV2": "npm:@maplelabs/debt-locker@v2.0.0",
+    "@maplelabs/debtLockerV3": "npm:@maplelabs/debt-locker@v3.0.0",
```
Comment (michaelsmueller) [inline package.json:41]:
```suggestion
    "@maplelabs/debtLockerV3": "npm:@maplelabs/debt-locker@v3.0.0-rc.0",
```

**Review by marko911 — APPROVED** [review]

**Review by adriamarti — APPROVED** [review]

**Pull request merged**